### PR TITLE
 Add orbit_uprobes_wine.py E2E test script

### PR DESCRIPTION
**Pull request by dpallotti**
This test verifies uprobes on `triangle.exe` running on Wine. It instruments two
functions, one from a Wine PE that has file alignment compatible with `mmap`'s
file mappings, and one from `triangle.exe`, which doesn't have file alignment
compatible with `mmap`'s file mappings.
We expect the first function to be recorded and the second one to have no hits,
but with a corresponding warning showing up in the Capture Log.

Bug: http://b/232072696

Test: Ran locally.

### CODE REVIEW

**Review by florian-kuebler — COMMENTED** [review]
Apart from the file name (and mentions in commit/pr message), lgtm.

**Comment by dpallotti** [comment]
> The test should be renamed (see offline discussion)

Thanks, done.

File: contrib/automation_tests/orbit_uprobes_wine.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        WaitForLoadingSymbolsAndCheckModule(module_search_string="triangle.exe"),
+        FilterAndHookFunction(
+            function_search_string="dxvk::vk::Presenter::presentImage{ }d3d11.dll"),
+        FilterAndHookFunction(function_search_string="Render{ }triangle.exe"),
+        Capture(user_space_instrumentation=False),
+        CheckTimers(track_name_filter="dxvk-submit"),
+        VerifyScopeTypeAndHitCount(scope_name="dxvk::vk::Presenter::presentImage",
+                                   scope_type="D",
+                                   min_hits=1000,
+                                   max_hits=15000),
```
Comment (danielfenner) [inline contrib/automation_tests/orbit_uprobes_wine.py:49]: Consider being even more generous with the bounds (I usually went with an  0.1 x  average, 10x average).

**Review by danielfenner — APPROVED** [review]
lgtm

File: contrib/automation_tests/orbit_uprobes_wine.py (hunk 1 above)
Comment (dpallotti) [reply contrib/automation_tests/orbit_uprobes_wine.py]: Sounds good. I also updated them in `orbit_user_space_instrumentation_silenus.py`, which is where I had copied these values from.

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks

**Pull request merged**